Datagrid/Pager.php

```diff
@@ -260,12 +276,29 @@ public function getFirstIndice()
         return ($this->page - 1) * $this->maxPerPage + 1;
     }
 
+
+    /**
+     * NEXT_MAJOR: remove this method
+     *
+     * @deprecated
```

Comment:
Please add a message to the `@deprecated` annoation like we do for any other method